docs/vector/getting_started.rst

```diff
@@ -0,0 +1,188 @@
+    ... }
+    >>> observations, rewards, dones, infos = envs.step(actions)
+    >>> observations
+    {"position": array([[-0.5337036 ,  0.7439302 ,  0.41748118],
+                        [ 0.9373266 , -0.5780453 ,  0.8987405 ],
+                        [-0.917269  , -0.5888639 ,  0.812942  ]], dtype=float32),
+    "velocity": array([[ 0.23626241, -0.0616814 ],
+                       [-0.4057572 , -0.4875375 ],
+                       [ 0.26341468,  0.72282314]], dtype=float32)}
+
+.. note::
+    The sub-environments inside a vectorized environment automatically call :obj:`reset` at the end of an episode. In the following example, the episode of the 3rd sub-environment ends after 2 steps (the agent fell in a hole), and the sub-environment gets reset (observation ``0``).
+
+    .. code-block::
+
+        >>> envs = gym.vector.make("FrozenLake-v1", num_envs=3, is_slippery=False)
+        >>> envs.reset()
+        array([0, 0, 0])
+        >>> observations, rewards, dones, infos = envs.step(np.array([1, 2, 2]))
+        >>> observations, rewards, dones, infos = envs.step(np.array([1, 2, 1]))
+
+        >>> dones
+        array([False, False,  True])
+        >>> observations
+        array([8, 2, 0])
+
+Observation & Action spaces
+---------------------------
+
+Like any Gym environment, vectorized environments contain two properties :attr:`~VectorEnv.observation_space` and :attr:`~VectorEnv.action_space` to specify the observation and action spaces of the environment. Since vectorized environments operate on multiple sub-environments, where the observations and actions of sub-environments are batched together, the observation and action spaces are adequately batched as well so that the input actions are valid elements of :attr:`~VectorEnv.action_space`, and the observations are valid elements of :attr:`~VectorEnv.observation_space`.
+
+.. code-block::
+
+    >>> envs = gym.vector.make("CartPole-v1", num_envs=3)
+    >>> envs.observation_space
+    Box([[-4.8 ...]], [[4.8 ...]], (3, 4), float32)
+    >>> envs.action_space
+    MultiDiscrete([2 2 2])
+
+.. note::
+    In order to appropriately batch the observations and actions in vectorized environments, the observation and action spaces of all the sub-environments are required to be identical.
+
+    .. code-block::
+
+        >>> envs = gym.vector.AsyncVectorEnv([
+        ...     lambda: gym.make("CartPole-v1"),
+        ...     lambda: gym.make("MountainCar-v0")
+        ... ])
+        RuntimeError: Some environments have an observation space different from `Box([-4.8 ...], [4.8 ...], (4,), float32)`. In order to batch observations, the observation spaces from all environments must be equal.
+
+However, sometimes it may be handy to have access to the observation and action spaces of a sub-environment, and not the batched spaces. You can access those with the properties :attr:`~VectorEnv.single_observation_space` and :attr:`~VectorEnv.single_action_space` of the vectorized environment.
```

Comment:
Thank you!! This is super helpful. I'd suggest highlighting this section with a heading like 

## Working with the spaces of a sub-environment